src/IBusBM.cpp

```diff
@@ -102,13 +102,16 @@ void IBusBM::begin(HardwareSerial &serial, int8_t timerid, int8_t rxPin, int8_t
   this->len = 0;
   this->chksum = 0;
   this->lchksum = 0;
+  for (uint8_t i = 0; i < PROTOCOL_CHANNELS; i++) {
+    channel[i] = 1500;
```

Comment:
Might initialize to 0? That should get flagged in `RemoteControl`, right?

Reply:
Mmm. Fair enough - though it won't be PWM's that we should be watching. It'll be a Serial message.

Reply:
Although now that I think about it discarding out-of-range speed settings could let the bot run away at the last valid setting, but that is a topic for the other PR.